proto/canto/erc20/v1/tx.proto

```diff
@@ -38,11 +61,15 @@ message MsgConvertCoinResponse {}
 // MsgConvertERC20 defines a Msg to convert a ERC20 token to a native Cosmos
 // coin.
 message MsgConvertERC20 {
+  //  option (cosmos.msg.v1.signer) = "sender";
```

Comment:
In MsgConvertErc20, `sender` is `evm address`, so adding option `(cosmos.msg.v1.signer) = "sender";` will cause the signer to use the `evm address` instead of the `cosmos AccAddress`. Therefore, for MsgConvertErc20, we need the `GetSignersFromMsgConvertERC20V2` function to customise this.